bluetooth/src/commonMain/kotlin/BluetoothMonitor.kt

```diff
@@ -20,6 +20,11 @@ package com.splendo.kaluga.bluetooth
 import kotlinx.coroutines.flow.StateFlow
 
 expect class BluetoothMonitor {
+
+    class Builder constructor() {
+        fun create(): BluetoothMonitor
```

Comment:
Shouldn't create method take parameter `bluetoothAdapter: BluetoothAdapter = BluetoothAdapter.getDefaultAdapter()` ?

Reply:
common does not know what does those things are